beacon_node/execution_layer/src/engine_api/json_structures.rs

```diff
@@ -400,75 +524,3 @@ pub mod serde_logs_bloom {
             .map_err(|e| serde::de::Error::custom(format!("invalid logs bloom: {:?}", e)))
     }
 }
-
```

Comment:
This is no longer necessary since we removed the union from the transaction type in the consensus spec

Reply:
Nice